submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles-menu.css" />
+    <title>HTML & CSS практика: Hooli-style Popup</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <header class="header">
+        <div class="logo">LOGO</div>
+        <nav class="navigation">
```

Comment:
When you have nested navigation better to use HTML lists for the first and other levels.